client/a8c-for-agencies/sections/purchases/licenses/revoke-license-dialog/style.scss

```diff
@@ -0,0 +1,113 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.revoke-license-dialog {
+	&.dialog.card {
+		max-height: 100vh;
+		max-width: 100vh;
+		margin: 0;
+		align-self: stretch;
+
+
+		@include break-small {
+			max-height: 90%;
+			max-width: 90%;
+			margin: auto 0;
+			align-self: center;
+		}
+	}
+
+	.dialog__content {
+		max-width: 628px;
+		align-self: stretch;
+		height: 100%;
+
+		@include break-small {
+			align-self: center;
+		}
+
+		&,
+		& p {
+			font-size: 1rem;
+		}
+	}
+
+	.dialog__action-buttons {
+		display: flex;
+		justify-content: flex-end;
+		align-items: stretch;
+		background: var(--studio-gray-0);
```

Comment:
Can we change the colors to use theme properties?

Reply:
Thanks for the review, @jkguidaven! Sorry for missing this. I have updated it: https://github.com/Automattic/wp-calypso/pull/88011/commits/94b63f87801d10f5e564c60a38116d4d1a1ac96f